### DISCUSSION TITLE
Add Hover for menu items?

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [elchininet/custom-sidebar#201] [Q&A]
thanks for the sidebar border color ;-) 

would you consider adding an option to set a hover color?
doing this myself:

```
      /*a:hover paper-icon-item ha-icon {
        color: var(--alert-color) !important;
      }*/
      /*a:hover paper-icon-item .item-text{
        color: var(--primary-color);
      }*/
      a:hover paper-icon-item {
        background: var(--hover-color);
      }
```

where the hover color is defined in the themes (in this case a gradient that I also use as my view background, just to show the possibilities)

<img width="273" alt="Scherm­afbeelding 2024-11-13 om 11 35 05" src="https://github.com/user-attachments/assets/315699bc-7626-4442-a758-bc954ccc8ac8">

**Comment by elchininet** [comment]
@Mariusthvdb,
Yes, it is possible to add more options to cover those.

**Reply by elchininet** [reply]
Hi @Mariusthvdb,
Check [this release](https://github.com/elchininet/custom-sidebar/releases/tag/v7.0.0). It contains this feature. As it is a major refactoring of the code, it would be helpful that you test it with your set up and report back if you notice something broken or working as not expected.
